desktop/core/src/desktop/js/reactComponents/FileChooser/FileChooserModal/FileChooserModal.scss

```diff
@@ -16,28 +16,40 @@
 
 @import '../../../components/styles/colors';
 
-.file-chooser__modal .ant-modal-body {
+.hue-file-chooser__modal .ant-modal-body {
   padding: 0;
   height: 554px;
 }
 
-.file-system__panel.ant-menu {
+.hue-file-chooser-modal__spinner.ant-spin-nested-loading {
```

Comment:
we should be able to size the any spinner using component parameters. Can we check this before customizing it.
https://ant.design/components/spin